jina/schemas/helper.py

```diff
@@ -10,7 +10,7 @@ def _python_type_to_schema_type(p):
     elif p == 'dict':
         dtype = 'object'
     else:
-        dtype = None
+        dtype = 'null'
```

Comment:
what is this?

Reply:
The `tests/unit/schemas/test_schema.py` test was failing because None is not a valid json data type. It might have been enforced in the latest version.